### PR TITLE
Improve tooltips 874

### PR DESCRIPTION
**Pull request by joeyzhaozhao**
## Description
Improve tooltips, checking ticket link 
https://github.com/Kwenta/kwenta/issues/874

## Motivation and Context
Improve serval components of tooltips, and remove hovertransferform from code

## How Has This Been Tested?
Check the issue above

### CODE REVIEW

File: sections/futures/TradingHistory/SkewInfo.tsx [hunk 1]
```diff
@@ -50,19 +100,47 @@ const SkewInfo: React.FC<SkewInfoProps> = ({ currencyKey }) => {
...
 
+const WithCursor = styled.div<{ cursor: 'help' }>`
+	cursor: ${(props) => props.cursor};
+`;
+
+const OpenInterestRow = styled.div`
+	display: flex;
+	width: 100%;
+	justify-content: space-between;
+	line-height: 16px;
```
Comment (LeifuChen) [inline sections/futures/TradingHistory/SkewInfo.tsx:128]: The numeric value needs to show as `mono` font.

**Review by LeifuChen — CHANGES_REQUESTED** [review]
@joeyzhaozhao Thanks for the contribution. Just leave minor comments for your reference.

File: sections/futures/PositionCard/PositionCard.tsx [hunk 2]
```diff
@@ -379,6 +352,7 @@ const StyledSubtitle = styled.p`
 	color: ${(props) => props.theme.colors.common.secondaryGray};
 	text-transform: capitalize;
 	margin: 0;
+	cursor: help;
```
Comment (LeifuChen) [inline sections/futures/PositionCard/PositionCard.tsx:355]: As @platschi mentioned, there is no need to include the price with the tooltip cursor.

**Review by LeifuChen — CHANGES_REQUESTED** [review]
@joeyzhaozhao Just found another tiny thing. As the default font of `tooltip` changed to `regular`, the update of the numeric value will lead to the change of the `TimerTooltip` 's width (which is set to `auto`). Could you please update the font of the numeric value to `mono` again to make sure all the numeric value has the same width? The change is supposed to be made in `TimerTooltip.tsx`. Thanks.
image

File: components/Tooltip/TimerTooltip.tsx [hunk 3]
```diff
@@ -3,6 +3,7 @@ import { Tooltip, ToolTipWrapper } from './TooltipStyles';
 import styled from 'styled-components';
 import { useTranslation } from 'react-i18next';
 import useInterval from 'hooks/useInterval';
+import { NumericValue } from 'styles/common';
```
Comment (LeifuChen) [inline components/Tooltip/TimerTooltip.tsx:6]: just remove this unnecessary import.

**Review by LeifuChen — APPROVED** [review]
@joeyzhaozhao Great job!

**Review by platschi — APPROVED** [review]
Great job, looks much better now!

**Pull request merged**